client/components/deposits-overview/utils.ts

```diff
@@ -1,7 +1,16 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { sprintf } from '@wordpress/i18n';
+import moment from 'moment';
+import interpolateComponents from '@automattic/interpolate-components';
+
 /**
  * Internal dependencies
  */
 import { formatCurrency } from 'utils/currency';
+import strings from './strings';
```

Comment:
Nit: There are some unused imports in this file.

Reply:
Oh thanks. I've fixed that in [42b629e](https://github.com/Automattic/woocommerce-payments/pull/5891/commits/42b629e7a729ae3c18bac097c0e9761917c22d9c)